src/Files/Helpers/ListedItemHelpers.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Files.Filesystem;
+
+namespace Files.Helpers
+{
+    public static class ListedItemHelpers
+    {
+        /// <summary>
+        /// Check if the <see cref="ListedItem"/> is a image file.
+        /// </summary>
+        /// <param name="listedItem">The <see cref="ListedItem"/> to check the file extension of.</param>
+        /// <returns><c>true</c> if the <see cref="ListedItem"/> is an image, otherwise <c>false</c></returns>
+        public static bool IsImage(this ListedItem listedItem)
+        {
+            if (listedItem is null)
+            {
+                return false;
+            }
+
+            if (string.IsNullOrEmpty(listedItem.FileExtension))
+            {
+                return false;
+            }
+
+            return listedItem.FileExtension.Equals(".png", StringComparison.OrdinalIgnoreCase) || 
```

Comment:
Instead of passing in a listed item, can this be simplified to passing in a file extension? This will make the helper more flexible and let us check the extension even when the file isn't a listed item.

Reply:
Sure, can do. That makes sense.

Reply:
I've added the helper. Should I go an update the other properties in the viewmodel that uses the extension checks? or leave that for another PR.

Reply:
I'm going to push a small change first and then you can apply it to the other places so that it's consistent